python/cudf/cudf/_lib/nvtx/_lib/lib.pyx

```diff
@@ -0,0 +1,64 @@
+from nvtx._lib.lib cimport *
+
+
+def initialize():
+    nvtxInitialize(NULL)
+
+
+cdef class EventAttributes:
+    cdef dict __dict__
+    cdef nvtxEventAttributes_t c_obj
+
+    def __init__(self, message=None, color=None):
+        self._message = message.encode("ascii")
```

Comment:
Is this a safe assumption? I'm imagining chinese characters for example. Should we consider using unicode instead?